community/modules/scheduler/pbspro-server/README.md

```diff
@@ -0,0 +1,126 @@
+## Description
+
+This module provisions a PBS Server Host to operate and administer a PBS
+Professional cluster. The following extra services are required:
+
+- an existing Altair license server with sufficient licenses to run PBS Pro
+- a shared filesystem mounted across all PBS hosts to faciliate file transfers
+  for jobs and their stdin/stdout logs
```

Comment:
No. The configuration magic occurs in the pbspro-execution module (forthcoming) that enables the execution host to recognize the filesystems as shared among all hosts and use `cp` (literally) to copy files instead of `scp` or `rcp`.

I believe the requirement could be stated as "jobs must be submitted from a shared filesystem..." followed by wording similar to above. Sound OK?

Reply:
Done on HEAD of PR